src/schema/rules/sidecars/eeg.yaml

```diff
@@ -151,16 +151,23 @@ EEGCoordsystemLandmark:
             level: recommended
             description_addendum: Preferably the same as the `EEGCoordinateSystem`.
         AnatomicalLandmarkCoordinateUnits: recommended
+
+EEGCoordsystemLandmarkDescriptionRec:
+    selectors:
+    - modality == "eeg"
+    - datatype == "eeg"
+    - suffix == "coordsystem"
+    - sidecar.AnatomicalLandmarkCoordinateSystem != "Other"
+    fields:
         AnatomicalLandmarkCoordinateSystemDescription:
             level: recommended
             level_addendum: required if `AnatomicalLandmarkCoordinateSystem` is `"Other"`
```

Comment:
I think you can drop the `level_addendum` now that the selectors don't include when `AnatomicalLandmarkCoordinateSystem` is `"Other"`.

Reply:
With the work @effigies is doing in the make table PR this level_addendum will be used in the table, but not for validation purposes.

Reply:
:+1: That makes sense.